modules/passkey/src/deploy/4337.ts

```diff
@@ -0,0 +1,24 @@
+import EntryPoint from '@account-abstraction/contracts/artifacts/EntryPoint.json'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { ENTRY_POINT } from '../constants'
+
+const deploy: DeployFunction = async ({ deployments, getNamedAccounts, network }) => {
+  const { deployer } = await getNamedAccounts()
+  const { deploy } = deployments
+
+  if (network.tags.dev || network.tags.test) {
+    await deploy('EntryPoint', {
+      from: deployer,
+      contract: EntryPoint,
+      args: [],
+      log: true,
+      deterministicDeployment: '0x90d8084deab30c2a37c45e8d47f49f2f7965183cb6990a98943ef94940681de3',
+    })
+  } else if (!ENTRY_POINT) {
+    throw new Error('TESTS_4337_MODULE_DEPLOYMENT_ENTRY_POINT_ADDRESS must be set')
```

Comment:
Since this is for testing, do we ever even need this? Might be simpler to always deploy the EntryPoint.

Reply:
Oh... we need it for the launchpad contract. In that case, I think it should be `DEPLOYMENT_ENTRY_POINT`, since when we deploy it for real on Polygon for example, it feels weird to use a `TEST_*` variable.

Reply:
good idea, removed

Reply:
Ah, I didn't read the second comment :facepalm: will restore it